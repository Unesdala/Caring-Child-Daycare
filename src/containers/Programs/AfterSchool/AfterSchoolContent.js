import React from 'react';

const AfterSchoolContent = () => (
  <main className="after-school">
    <p />
  </main>
);

export default AfterSchoolContent;
