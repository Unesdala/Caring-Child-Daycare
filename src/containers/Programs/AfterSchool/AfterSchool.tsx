import AfterSchoolContent from './AfterSchoolContent';

export const AfterSchool = (): JSX.Element => {
  document.title = 'Caring Child Daycare | After School';  
  return (
    <div className="Site-content">
      <main className="after-school-main">
        <AfterSchoolContent />
      </main>
    </div>
  );
};

export default AfterSchool;
