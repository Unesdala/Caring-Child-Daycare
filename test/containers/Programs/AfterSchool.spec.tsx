/* eslint-disable @typescript-eslint/no-explicit-any */

import { shallow } from 'enzyme';
import { AfterSchool } from 'src/containers/Programs/AfterSchool/AfterSchool';
import AfterSchoolContent from 'src/containers/Programs/AfterSchool/AfterSchoolContent';

describe('Afterschool', () => {
  const wrapper = shallow(<AfterSchool />);
  it('renders', () => {
    expect(wrapper.find(AfterSchoolContent).exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find(AfterSchoolContent).dive().find('main.after-school-main--content').exists()).toBe(true);
  });
  
});
