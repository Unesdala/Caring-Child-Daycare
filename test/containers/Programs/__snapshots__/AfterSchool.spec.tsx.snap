// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Afterschool renders 1`] = `ShallowWrapper {}`;
